.github/workflows/dependencies.yml

```diff
@@ -1,17 +1,11 @@
 name: dependencies
 
 on:
+  schedule:
+    - cron: '0 21 * * TUE' # Run every Tuesday at 21:00 (UTC)
   push:
-    branches:
-      - main
-    paths:
-      - '**/Cargo.lock'
-      - '**/Cargo.toml'
-  pull_request:
-    types: [opened, synchronize]
-    paths:
-      - '**/Cargo.lock'
-      - '**/Cargo.toml'
+    tags:
+      - 'v*.*.*'
```

Comment:
Not sure how this will impact the dependency checks? Is it going to run everytime we push a new release? Ultimately, it would be cool to check de dependencies _before_ doing a release.

Reply:
Exactly, it will **also** run when we push a new release. Thus, it will run every Tuesday at 21:00 (UTC) **and** when we push a new release.